fix(deps): update prisma monorepo to v4 (major) (#7671)
keystonejs/keystone: .changeset/slow-games-greet.md

---
'@keystone-6/core': major
---

> Returning `null` from a `resolveInput` hook for json fields will now store a database null as expected

Reviewer: ```suggestion
Changes the return type for `resolveInput` hook for `json` fields.  Previously you used `'DbNull'` or `'JsonNull'` for respective null magic values - you can now use a Javascript `null` value which is mapped by Keystone to a `Prisma.DbNull`.
```

> Changes the return type for `resolveInput` hook for `json` fields.  Previously you used `'DbNull'` or `'JsonNull'` for respective null magic values - you can now use a Javascript `null` value which is mapped by Keystone to a `Prisma.DbNull`.

Reviewer: ```suggestion
Changes the return type for `resolveInput` hook for `json` fields.  Previously you may have used `'DbNull'` or `'JsonNull'` for respective null magic values - you can now always use the Javascript `null` value, as it will always mapped by Keystone to a `Prisma.DbNull`.
```
